site/en/docs/extensions/migrating/mv2-sunset/index.md

```diff
@@ -5,17 +5,18 @@ subhead: 'Understand when Manifest V2 will stop working for extensions'
 description: 'Details of the Manifest V2 phase-out and end of life.'
 layout: 'layouts/doc-post.njk'
 date: 2021-09-23
-updated: 2022-12-13
+updated: 2023-06-01
 tags:
   - extensions-news
 ---
 
-As Manifest V3 approaches full feature parity with Manifest V2, we'll progressively phase out Manifest V2.
-
 We're still working on the timeline of the MV2 phase-out plan, so keep an eye out for it in the
 coming months. We will provide at least 6 months between a timeline announcement and any experiments
 deprecating MV2 features.
 
+Before announcing a new timeline, we will address prioritized platform gaps and close critical bugs.
```

Comment:
Nit: “we are addressing” is better than “will address”

Reply:
What about:

"We are addressing prioritized platform gaps and closing critical bugs before announcing a new timeline."

Reply:
I've pushed a tweak - let me know if that's ok. I kept "Before announcing a new timeline" at the beginning since I think that's important to emphasize that we see it as a prerequisite.